app/src/main/res/layout/fragment_home.xml

```diff
@@ -88,22 +103,40 @@
                     android:id="@+id/iv_following_header"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:layout_marginStart="12dp"
+                    android:layout_marginLeft="12dp"
                     android:layout_marginTop="36dp"
                     android:contentDescription="@string/description_following_header"
-                    app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toBottomOf="@+id/rv_big_read"
+                    app:layout_constraintLeft_toLeftOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/shimmer_big_read"
                     app:srcCompat="@drawable/ic_following_text"
                     tools:srcCompat="@drawable/ic_following_text" />
 
+                <com.facebook.shimmer.ShimmerFrameLayout
+                    android:id="@+id/shimmer_following"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    app:shimmer_repeat_mode="restart"
+                    app:layout_constraintTop_toBottomOf="@id/iv_following_header"
+                    app:shimmer_shape="linear">
+                    <LinearLayout
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:orientation="vertical">
+                        <include layout="@layout/item_article_placeholder"></include>
```

Comment:
what do you think about naming these to layout_shimmer_article for consistency with the naming of layout_shimmer_big_read, or maybe layout_shimmer_article_placeholder and layout_shimmer_big_read_placeholder for these to not be mistaken as the main shimmerframelayout